Exec/science/nova/inputs_nova_t7

```diff
@@ -69,36 +113,45 @@ amr.v                 = 1       # verbosity in Amr.cpp
 
 
 # REFINEMENT / REGRIDDING
-amr.max_level       = 2            # maximum level number allowed
+amr.max_level       = 1            # maximum level number allowed
 amr.ref_ratio       = 4 2          # refinement ratio
-amr.regrid_int      = 2 2          # how often to regrid
-amr.max_grid_size   = 256
-amr.blocking_factor = 64
-amr.n_error_buf     = 2 2         # number of buffer cells in error est
-amr.refine_grid_layout = 0
-
+amr.regrid_int      = 5000         # how often to regrid
```

Comment:
really? why do you set this?

Reply:
I took it from the masive star input, and i though that reducing how ofter we regrid we can reduce the number of memory operations.

Reply:
I would leave that out for now.  We will want to regrid more often as the nova envelope expands.